exercises/practice/dnd-character/.meta/src/example.cr

```diff
@@ -0,0 +1,32 @@
+class DndCharacter
+  property name : String
+  property strength : Int32
+  property dexterity : Int32
+  property constitution : Int32
+  property intelligence : Int32
+  property wisdom : Int32
+  property charisma : Int32
+  property hitpoints : Int32
+
+  def initialize(name : String = "Blandalf the Beige")
+    @name = name
```

Comment:
Looks like `@name` is unused and unneeded. My vote is to remove it to avoid confusion.